capture-sdk/sdk/src/test/java/net/gini/android/capture/analysis/AnalysisFragmentTest.kt

```diff
@@ -33,7 +35,8 @@ class AnalysisFragmentTest {
     fun `triggers Cancel event when back was pressed`() {
         // Given
         val eventTracker = spy<EventTracker>()
-        GiniCapture.Builder().setEventTracker(eventTracker).build()
+        GiniCapture.newInstance(InstrumentationRegistry.getInstrumentation().context)
+            .setEventTracker(eventTracker).build()
```

Comment:
@a-szotyori attention on this line, please
I changed it because since I implemented initializion of analytics at `GiniCapture.initialize(context)` we need to pass context for correct initialization. 
For situations, when someone uses deprecated `GiniCapture.newInstance()` without context I added extra initialization at first screen (`CameraFragment`)